libs/wingsdk/src/platform/platform-manager.ts

```diff
@@ -232,9 +232,11 @@ export function _loadCustomPlatform(customPlatformPath: string): any {
     script.runInContext(context);
     return new (platformModule.exports as any).Platform();
   } catch (error) {
+    const hint = isScoped
+      ? `Ensure you have installed the platform package by running 'npm install ${customPlatformPath}'`
+      : "Ensure the path to the platform is correct";
     console.error(
-      "An error occurred while loading the custom platform:",
-      error
+      `An error occurred while loading the custom platform: ${customPlatformPath}\n\n(hint: ${hint})`
```

Comment:
I’m not sure how we could make using a non-existent platform clearer. In the original you get a big gnarly stack trace. I assume with these changes it might be clearer that they are using the wrong platform name, since it’s less verbose

Reply:
Maybe instead of `isScoped`, check if the platform contains a `.`.
If it does, assume it's a path can give the path error. Otherwise give the error that the package should be installed

Reply:
Ah! Now I see the point you were making. 👍 Changed the check